README.md

```diff
@@ -430,21 +430,21 @@ Netdata is a "ready to use" monitoring solution. Prometheus and Grafana are tool
 <details><summary>Click to see detailed answer ...</summary>
 &nbsp;<br/>&nbsp;<br/>
 
-First we have to say that Prometheus as a time-series database and Grafana as a visualizer are excellent tools for what they do.
+First, we have to say that Prometheus as a time-series database and Grafana as a visualizer are excellent tools for what they do.
 
 However, we believe that such a setup is missing a key element: A Prometheus and Grafana setup assumes that you know everything about the metrics you collect and you understand deeply how they are structured, they should be queried and visualized.
 
-In reality this setup has a lot of problems. The vast number of technologies, operating systems, and applications we use in our modern stacks, makes it impossible for any single person to know and understand everything about anything. We get testimonials regularly from Netdata users across the biggest enterprises, that Netdata manages to reveal issues, anomalies and problems they were not aware of and they didn't even have the means to find or troubleshoot.
+In reality, this setup has a lot of problems. The vast number of technologies, operating systems, and applications we use in our modern stacks, makes it impossible for any single person to know and understand everything about anything. We get testimonials regularly from Netdata users across the biggest enterprises, that Netdata manages to reveal issues, anomalies and problems they were not aware of and they didn't even have the means to find or troubleshoot.
 
-So, the biggest difference of Netdata to Prometheus and Grafana, is that we decided that the tool needs to have a much better understanding of the components, the applications and the metrics it monitors.
+So, the biggest difference between Netdata Prometheus and Grafana, is that we decided that the tool needs to have a much better understanding of the components, the applications, and the metrics it monitors.
```

Comment:
```suggestion
So, the biggest difference between Netdata and Prometheus and Grafana, is that we decided that the tool needs to have a much better understanding of the components, the applications, and the metrics it monitors.
```

Reply:
Reverting my change, reading this again I prefer the original approach